ocs_ci/ocs/perftests.py

```diff
@@ -41,9 +42,46 @@ def setup(self):
         self.benchmark_obj = None  # place holder for the benchmark object
         self.client_pod = None  # Place holder for the client pod object
         self.dev_mode = config.RUN["cli_params"].get("dev_mode")
-        self.environment = get_environment_info()
         self.pod_obj = OCP(kind="pod")
 
+        # Collecting all Environment configuration Software & Hardware
+        # for the performance report.
+        self.environment = get_environment_info()
+        self.environment["clusterID"] = get_running_cluster_id()
+
+        ct_pod = pod.get_ceph_tools_pod()
+        osd_info = ct_pod.exec_ceph_cmd(ceph_cmd="ceph osd df")
+        self.environment["osd_size"] = osd_info.get("nodes")[0].get("crush_weight")
+        self.environment["osd_num"] = len(osd_info.get("nodes"))
+        self.environment["total_capacity"] = osd_info.get("summary").get(
+            "total_kb_avail"
+        )
+        self.environment["ocs_nodes_num"] = len(node.get_ocs_nodes())
+
+        oc_cmd = OCP(namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+
+        # Getting worker nodes information
```

Comment:
maybe separate functions for worker node info and maser node info too.

Reply:
done